sdk/appconfiguration/app-configuration/test/package.spec.ts

```diff
@@ -8,9 +7,19 @@ describe("packagejson related tests", () => {
   // if this test is failing you need to update the contant `packageVersion` referenced above
   // in the generated code.
   it("user agent string matches the package version", () => {
-    const packageJsonFilePath = join(__dirname, "../package.json");
-    const rawFileContents = readFileSync(packageJsonFilePath, { encoding: "utf-8" });
-    const packageJsonContents = JSON.parse(rawFileContents);
+    let packageJsonContents: {
+      [property: string]: string;
+    };
+
+    try {
+      // For integration tests
+      const packageJsonFilePath = join(__dirname, "../../package.json");
+      packageJsonContents = require(packageJsonFilePath);
+    } catch (e) {
+      // For unit tests
+      const packageJsonFilePath = join(__dirname, "../package.json");
```

Comment:
Why are these different?

Reply:
The integration test one has the test files in dist-esm/test/* while the unit test one runs from test/